src/containers/AboutUs/AboutUs.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { styled } from '../../styles';
+import { H1, H3 } from '../../components';
+import { AboutUsTeamImage } from '../../assets';
+
+const Wrapper = styled.div`
+  width: 60%;
+  height: 80%;
+  display: flex;
+  flex-direction: column;
+`;
+
+const TitleWrapper = styled(H1)`
+  font-size: 300%;
+  text-align: center;
+`;
+
+const ContentWrapper = styled.div`
+  width: 100%;
+  height: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const AboutUsWrapper = styled.div`
+  width: 600px;
+  height: 250px;
+  background-color: ${ ({theme}) => theme.colors.primary };
+  opacity: 0.65;
+  left: 30%;
+  position: absolute;
```

Comment:
bad styles. could you remove the absolute position
remove 'left' as well

would be better to use display flex here and padding